src/components/MintDialog/pages/Success/Success.tsx

```diff
@@ -59,6 +61,11 @@ export const Success: FC<SuccessProps> = ({
       <div>
         <h3 className="font-sans text-[16px] text-[#151515]">Share on </h3>
         <Share />
+        {mintType === MintType.ThirdWeb && (
+          <p className="text-sm font-mono text-[#5B616E] mt-6 flex items-center">
+            Mint contract powered by <ThirdWebPill className="inline ml-2" />
```

Comment:
could chuck a sr-only span in here saying "ThirdWeb"

```suggestion
            Mint contract powered by <span className="sr-only">ThirdWeb</span><ThirdWebPill className="inline ml-2" />
```